pkgs/by-name/wa/waywall/package.nix

```
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  egl-wayland,
+  libglvnd,
+  libspng,
+  libxkbcommon,
+  luajit,
+  mesa_glu,
+  meson,
+  ninja,
+  pkg-config,
+  wayland,
+  wayland-protocols,
+  wayland-scanner,
+  xorg,
+  xwayland,
+}:
+stdenv.mkDerivation {
+  pname = "waywall";
+  version = "0-unstable-2025-02-07";
+
+  src = fetchFromGitHub {
+    owner = "tesselslate";
+    repo = "waywall";
+    rev = "be96e20997c5886af9661d9832b7902aba1e5311";
+    hash = "sha256-77GbBzHjyZuauhl0vlguUS/7jBT4qNjOLYGWBVTPjEY=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    meson
+    ninja
+    pkg-config
+    wayland-scanner
```

```suggestion
    wayland-protocols
    wayland-scanner
```
Upstream uses [`wayland-protocols`](https://github.com/tesselslate/waywall/blob/be96e20997c5886af9661d9832b7902aba1e5311/meson.build#L34C33-L34C50)